packages/matter.js/src/session/pase/PaseServer.ts

```diff
@@ -46,13 +56,42 @@ export class PaseServer implements ProtocolHandler<MatterDevice> {
         try {
             await this.handlePairingRequest(exchange.session.getContext(), messenger);
         } catch (error) {
-            logger.error("An error occured during the commissioning", error);
-            await messenger.sendError();
+            this.pairingErrors++;
+            logger.error("An error occurred during the PASE commissioning.", error);
+
+            // if we got a ChannelStatusResponseError in we do not need to send the same our
+            const sendError = !(error instanceof ChannelStatusResponseError);
+            await this.cancelPairing(messenger, sendError);
+
+            if (this.pairingErrors >= PASE_COMMISSIONING_MAX_ERRORS) {
+                throw new MaximumPasePairingErrorsReachedError(
+                    `Pase server: Too many errors during PASE commissioning, aborting commissioning window`,
+                );
+            }
         }
     }
 
     private async handlePairingRequest(server: MatterDevice, messenger: PaseServerMessenger) {
-        logger.info(`Pase server: Received pairing request from ${messenger.getChannelName()}`);
+        // When a Commissioner is either in the process of establishing a PASE session with the Commissionee or has
+        // successfully established a session, the Commissionee SHALL NOT accept any more requests for new PASE
+        // sessions until session establishment fails or the successfully established PASE session is terminated on
+        // the commissioning channel.
+        if (server.existsOpenPaseSession()) {
+            throw new MatterFlowError(
```

Comment:
MatterFlowError is documented as more of an internal error (it says "please report these errors"), is there a better error class?  Should we be responding with a AdministratorCommissioning "Busy" status code?

Reply:
Terminology in chip "do not accept new connections" means ... answer nothing, let tthe other side timeout ... but right ... we also did in one other place where I send an error status back ... i do the same here. good idea.

Reply:
I checked ... the thrown connection already leads to an Status Report with InvalidParam.

In fact mor me "MatterFlowError" means that something happend which is not matching to the "expected flow of messages". So es it sy" please report" and in fact this is also true ... it shouldnot happen, but in this case it could when someone tries a commissioning when alreayd one is in progress. Butthere could also be cases where pase session is somehow blocked and then it makes sense to check. SO for me ok that way

Reply:
Is "Busy" status code appropriate though?

Reply:
Nvm, saw you already got it